src/cavendish_particle_tracks/_widget.py

```diff
@@ -404,13 +414,36 @@ def _on_click_load_data(self) -> None:
             self.msg.show()
             return
 
-        for subdir, stack_name in zip(
-            folder_subdirs, ["stack1", "stack2", "stack3"]
-        ):
-            stack = imread(subdir + "/*")
-            self.viewer.add_image(stack, name=stack_name)
-            # TODO: investigate the multiscale otption.
-        self.viewer.dims.axis_labels = ("Event", "Y", "X")
+        def crop(array):
+            # Crops view 1 and 2 to same size as view 3 by removing whitespace
+            # on left, as images align on the right.
+            return array[:, :, -8377:, :]
+
+        stacks = []
+        for subdir in folder_subdirs:
+            stack: da = imread(subdir + "/*")
+            if not self.test_mode:
+                stack = crop(stack)
+            stacks.append(stack)
+
+        # Concatenate stacks along new spatial dimension such that we have a view, and event slider
+        concatenated_stack = da.stack(stacks, axis=0)
+        self.viewer.add_image(concatenated_stack, name="Particle Tracks")
+        self.viewer.dims.axis_labels = ("View", "Event", "Y", "X")
+
+        measurement_layer_present = False
+        for layer in self.viewer.layers:
+            if layer.name == "Radii and Lengths":
+                measurement_layer_present = True
+                break
```

Comment:
```suggestion
        measurement_layer_present = "Radii and Lengths" in self.viewer.layers
```

Reply:
Although I might switch the logic to make a more readable if-statement...

```python
measurement_layer_name = "Radii and Lengths"
measurement_layer_not_present = measurement_layer_name not in self.viewer.layers
if measurement_layer_not_present:
    self.layer_measurements = self.viewer.add_points(
        name=measurement_layer_name,
        ...
    )
```

Reply:
> 

this works! implementing elsewhere. Guess it makes sense that the indexable/searchable key for the layer would be the name.

Reply:
> Although I might switch the logic to make a more readable if-statement...
> 
> ```python
> measurement_layer_name = "Radii and Lengths"
> measurement_layer_not_present = measurement_layer_name not in self.viewer.layers
> if measurement_layer_not_present:
>     self.layer_measurements = self.viewer.add_points(
>         name=measurement_layer_name,
>         ...
>     )
> ```

Much more readable. I'll implement this.